src/pkg/api/handlers.go

```diff
@@ -776,6 +776,21 @@ func getVirtualService(cache *resources.Cache) func(w http.ResponseWriter, r *ht
 	return rest.BindCustomResource(cache.VirtualServices, cache)
 }
 
+// @Description Get Gateways
+// @Tags gateways
```

Comment:
```suggestion
// @Tags networks
```
this will place the route under the networks section in the swagger docs